bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -752,7 +752,6 @@ impl Connection {
     /// client_hello.fingerprint_hash(FingerprintType::JA3, &mut hash);
     /// drop(conn);
     /// ```
-    #[cfg(feature = "unstable-fingerprint")]
     pub fn client_hello(&self) -> Result<&crate::client_hello::ClientHello, Error> {
```

Comment:
Was there any reason the client_hello method wasn't public? Did we have some concern around its safety or memory management?

Reply:
I'm guessing it's because is much cleaner than having to individually tag each fingerprint method: #4009. Maybe @jmayclin knows?

Reply:
I don't have an exact recollection, but I'd imagine that @maddeleine is exactly correct. This way we only have two `cfg` statements in total, one for this method and one for the top level `mod client_hello` in `lib.rs`.